examples/rest-v5-next-cursor.ts

```diff
@@ -0,0 +1,39 @@
+import { RestClientV5, UniversalTransferRecordV5 } from '../src/index';
+
+// or
+// import { RestClientV5 } from 'bybit-api';
+
+const client = new RestClientV5({
+  testnet: false,
+  key: 'insert_api_key',
+  secret: 'insert_api_secret',
+});
+
+async function getAllUniversalTransfers() {
+  let allTransfers: UniversalTransferRecordV5[] = [];
+  let nextCursor = '';
+  let pages = 0;
+
+  do {
+    pages++;
+    console.log(`Fetching data from page ${pages}`);
+    const response = await client.getUniversalTransferRecords({
+      limit: 50, // Maximum page size per request
+      cursor: nextCursor || undefined, // Only send cursor if we have one
+    });
+
+    if (response.result.list && response.result.list.length > 0) {
+      allTransfers = [...allTransfers, ...response.result.list];
```

Comment:
Instead of overwriting the array completely with each iteration, if you make this append-only, you can make allTransfers a const.
```suggestion
      allTransfers.push(...response.result.list);
```